<launch>
  <include file="$(find openni_launch)/launch/openni.launch" output="log"/>  

  <node name="kinect_capture_frame" pkg="kinect_capture_frame" type="kinect_snapshot_service" required="true" output="screen"/>

</launch>
